translations/pt-br/01-introdução.md

```diff
@@ -0,0 +1,71 @@
+# Introdução
+
+Bem-vindo ao curso "Getting Started with Open Source"!
+Este curso abrangente irá guiá-lo pelo fascinante mundo do open source e ajudá-lo a se tornar um colaborador ativo.
+Ao final deste curso, você deve ter uma compreensão sólida de como encontrar, participar e contribuir para projetos de open source, bem como desenvolver o seu currículo de open source. 
```

Comment:
Sorry I guess I'm missing context but "currículo de open source" doesn't make sense for me. Not even in the original. 

So as to not deviate from the original I'd not change here. IMO it should be "currículo de contribuições open source" or sometinh. But then it should come first from the source material I guess

Reply:
Hi @vasfvitor! :) This part of the introduction talks about the purpose of the course. In reading it in English, I do agree that the wording could be a bit better.